src/app/components/homePage/shared/checkAvailability.js

```diff
@@ -105,37 +112,56 @@ class CheckAvailability extends React.Component {
       return dateElements
     }
 
+    _changeAlert(value){
+      this.snackalert = value;
+      this.setState(() => {return true;})
+    }
+
+    _showSnackBar(){
+      var t = this.refs.snackbar
+        t.classList = "snackbar show";
+
+        return setTimeout(function(){ t.classList = "snackbar"; }, 2000);
+    }
+
     _handleDeleteTimeClick(time, date, index, formatDate, formatTime, event) {
       event.preventDefault()
       if (confirm(`Are you sure you want to delete times ${formatTime} on ${formatDate}?`))
       var request = new Api()
       request.deleteAvailability(userStore.personID, currentEvent.eventID, date, time).then((response) => {
         currentEvent.availability[date].splice(index, 1)
-        alert(`Successfully deleted times ${formatTime} on ${formatDate}`)
-        this.setState(() => {return true})
+        this._changeAlert(`Successfully deleted times ${formatTime} on ${formatDate}`)
+        this._showSnackBar()
+        //this.setState(() => {return true})
       }).catch((error) => {
-        alert('Could not delete the selected times, please try again, and if error persists contact the technical team.')
+        this._changeAlert('Could not delete the selected times, please try again, and if error persists contact the technical team.')
```

Comment:
For more text heavy errors we may want to also pass along a longer time to show the warning?